expr/expr.go

```diff
@@ -3613,6 +3613,48 @@ func EvalExpr(e *expr, from, until int32, values map[MetricRequest][]*MetricData
 
 		return results, nil
 
+	case "removeBetweenEpochs": // removeBetweenEpochs(seriesLists, fromEpoch, untilEpoch)
+		args, err := getSeriesArg(e.args[0], from, until, values)
+		if err != nil {
+			return nil, err
+		}
+
+		fromEpoch, err := getIntArg(e, 1)
```

Comment:
Very good suggestion, but the fact that dateParamToEpoch uses Config.defaultTimeZone forces us to make expr.go stateful. That implies that we have to convert some of its static functions into object methods.